docker/nginx/conf.d.templates/server.account.conf

```diff
@@ -9,6 +9,9 @@
 	server {
 		server_name account.{{PORTAL_DOMAIN}}; # example: account.siasky.net
 
+		set_by_lua_block $skynet_portal_domain { return "{{PORTAL_DOMAIN}}" }
+		set_by_lua_block $skynet_server_domain { return "{{SERVER_DOMAIN}}" == "" and "{{PORTAL_DOMAIN}}" or "{{SERVER_DOMAIN}}" }
```

Comment:
this is mixing `and` and `or` without brackets, I'm assuming it's correct so I'll approve but should we add them? could we add them even though they're implicit for whatever reason to improve readability

Reply:
changed in https://github.com/SkynetLabs/skynet-webportal/pull/1771/commits/520f935cd2d3ba7eb2dd1586e2681907111a3cc3